templates/_helpers.tpl

```diff
@@ -5,3 +5,11 @@
 {{ .Release.Name }}-secrets
 {{- end }}
 {{- end}}
+
+{{- define "kubectlVersion" }}
+{{- if .Values.global.kubectlVersion -}}
+{{ .Values.global.kubectlVersion }}
+{{- else -}}
+{{ printf "%s.%s" .Capabilities.KubeVersion.Major .Capabilities.KubeVersion.Minor }}
```

Comment:
I added `kubectlVersion` in global because this is not working with EKS versions, this is very strange because we have a `+` in the Minor

Reply:
Aaah that makes sense then. 

Well with this approach, providing the `global.kubectlVersion` will override the "default" of Capabilities being used, but if you'd prefer I can revert that bit and we can stick with what was there originally with a note it's recommended when enabling redis/postgres?

Reply:
No worries, we can go this way, I need to document this chart, so we can add a note around this